.github/workflows/tacos_apply.yml

```
@@ -91,6 +90,15 @@ jobs:
       ###     # TODO: github-script to fetch run-id of the most recent tfplan
       ###     run-id: ${{steps.find-plan.output.run-id}}
 
+      - name: get reviews
+        uses: buildsville/review-summary@v1.0.1
+        id: reviews
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: set reviews variable
+        run: gha-set-env 'GITHUB_PR_APPROVALS' echo ${{steps.reviews.outputs.approved}}
```

I tried to read the source code of that action, and it's fully unintelligible:
https://github.com/buildsville/review-summary/blob/master/index.js#L11-L37

A brief inspection shows that the necessary data is available via gh CLI. Prefer that, as it's much easier to troubleshoot via our logging -- we can copy-paste the bits of the command to see whether they get the right value.

https://gist.github.com/bukzor/32840ac573137b48a4ceb47325e431ac

I modified the var name to better match its content. Completely untested:

```suggestion
      - name: set $GITHUB_PR_APPROVERS
        run: |
          # value is prettified listing of approvers or empty-string
          
          # get all reviews  #FIXME: need a variable for the PR
          gh pr view https://github.com/getsentry/tacos-gha/pull/85 --json reviews |
            # keep only the approving reviews
            jq '.reviews | map(select(.state == "APPROVED"))' |
            # prettify
            jq 'map(.author.login) | join(", ")' |
            gha-set-env 'GITHUB_PR_APPROVERS' cat \
          ;
```